js/components/portal-dashboard/account-owner.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import css from "../../../css/portal-dashboard/header.less";
+
+export class AccountOwnerContainer extends React.PureComponent {
+    render() {
+        return (
+            <div className={css.accountOwner} data-cy="account-owner">
+                <div className={css.accountOwnerName}>Kristen Teachername</div>
+                <div className={css.accountOwnerIcon}>
+                    {this.renderIcon(`${css.icon}`, "#icon-account-owner")}
+                </div>
+            </div>
+        );
+    }
+    private renderIcon = (cssClass: string, iconId: string) => {
```

Comment:
We could easily refactor this into an Icon React functional component that takes in two props (a CSS class and an icon id - both strings) to eliminate this Code Climate complaint.

Reply:
Although with the CSS modules used in this project I'm not sure if passing the CSS class is anti-pattern.